d2l-table-wrapper.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-scroll-wrapper.html">
+
+<!--
+# D2L Table Wrapper
+
+Not meant to be used directly. Automatically created when using `<table is="d2l-table">`
+-->
+<dom-module id="d2l-table-wrapper">
+	<template strip-whitespace>
+		<style>
+			:host {
+				background-color: transparent;
+				display: block;
+				width: 100%;
+				position: relative;
+
+				--d2l-table-border-overflow: dashed 1px #d3d9e3;
+			}
+
+			d2l-scroll-wrapper {
```

Comment:
I see, so these styles only apply in this instance, not in general to all `d2l-scroll-wrapper`s, gotcha.